blocks/research-publications-cards/research-publications-cards.css

```diff
@@ -0,0 +1,158 @@
+.title-button-wrapper{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 50px;
+
+    .title{
```

Comment:
Can you format your entire CSS file once. Indentation and bracket spacing looks off.